packages/app/src/app/overmind/factories.ts

```diff
@@ -127,6 +128,17 @@ export const withOwnedSandbox = <I>(
           sandboxId: sandbox.id,
         });
       } catch (e) {
+        const isAnonServerForkMessage = e.message.includes(
```

Comment:
nitpick: Can we use error codes for this?

Reply:
Yeah I was thinking the same thing. I remember this was done somewhere else so I went with it, but since we're not using the message at all we can definitely use error codes. Do we have a way to generate these somehow or can I just come up with one?